Diy (#30)
AfroPython/Manual-do-evento: diy/extras.md

# Extras

> Não é sempre que se tem a grana para fazer camisas do AfroPython com aquelas capulanas maneiras para todas(os) participantes. Nós temos alguma dicas para preparar algo legal para participantes e treinadoras(es) sem precisar gastar uma fortuna nisso.

Reviewer: ```suggestion
Nós sabemos que todos querem ter uma camiseta do AfroPython, mas também sabemos que não é sempre que se tem a grana para fazer nossas camisetas com aquelas capulanas maneiras para todas(os) participantes. Nós temos alguma dicas para preparar algo legal para participantes e treinadoras(es) sem precisar gastar uma fortuna nisso.
```

Acho interessante adicionar o link para o que é uma capulana.

> - Em geral, pedir sacolas ecológicas são mais baratas do que pedir camisas, caso não se tenha grana para isso. De qualquer forma tome cuidado para que seja um produto que não estrague fácil antes de pedir um monte deles. Mande um e-mail para o AfroPython para pedir o modelo que usamos em eventos passados.

Reviewer: ```suggestion
- Em geral, pedir sacolas ecológicas são mais baratas do que pedir camisas, mas de qualquer forma tome cuidado para que seja um produto de qualidade antes de realizar o pedido. Nós já temos um modelo pronto para essas sacolas, sinta-se a vontade para mandar um e-mail para o AfroPython pedindo o modelo que usamos em eventos passados, mas também sinta-se a vontade criar novos modelos, lembrando sempre de respeitar as regras dos logos que estão descritas aqui neste manual.
```

> - Se essas opções ainda forem mais caras do que o orçamento permite, pode se preparar pacotes pequeno com os brindes que foram oferecidos pelos patrocinadores. Algumas outras coisinhas com adesivos e bilhetes escritos a mão podem ser um ótimo presente, algo como "Muito obrigado por vir! Ubuntu!".

Reviewer: ```suggestion
- Se essas opções ainda forem mais caras do que o orçamento permite, podem se preparar pequenos pacotes  brindes que foram oferecidos pelos patrocinadores. Algumas outras coisinhas com adesivos e bilhetes escritos a mão podem ser um ótimo presente, algo como "Muito obrigado por vir! Ubuntu!".
```

> Não se esqueça o objetivo não é ter o mais caro e impressionante brindes para todo mundo mas sim mostrar para as pessoas como elas são importantes e nos importamos com ela. Apenas para fazer as pessoas se sentirem especiais como são.\

Reviewer: ```suggestion
Não se esqueça o objetivo não é ter o mais caro e impressionante brinde para todo mundo mas sim mostrar para as pessoas como elas são importantes e nos importamos com ela. Apenas para fazer as pessoas se sentirem especiais como são.
```
